mypy/stubdoc.py

```diff
@@ -325,7 +325,7 @@ def args_kwargs(signature: FunctionSig) -> bool:
             return has_arg("*args", signature) and has_arg("**kwargs", signature)
 
         # Move functions with (*args, **kwargs) in their signature to last place.
-        return list(sorted(self.signatures, key=lambda x: 1 if args_kwargs(x) else 0))
+        return sorted(self.signatures, key=lambda x: 1 if args_kwargs(x) else 0)
```

Comment:
```suggestion
        return sorted(self.signatures, key=lambda x: bool(args_kwargs(x)))
```

Reply:
I left this one alone because it's not the most obvious what the sort order of booleans is

Reply:
Fair